main.js

```diff
@@ -0,0 +1,235 @@
+});
+//sets player o name
+player2Button.addEventListener('click', () => {
+	playerOName.textContent = names.value;
+	names.value = '';
+});
+//sets up settings for a player vs player match
+pVPButton.addEventListener('click', () => {
+	pVCButton.disabled = true;
+	pVPButton.disabled = true;
+	pVPButton.className = 'selectedButton';
+});
+//sets up setting for a player vs computer match
+pVCButton.addEventListener('click', () => {
+	pVCButton.disabled = true;
+	pVPButton.disabled = true;
+	player2Button.disabled = true;
+	computer = true;
+	playerOName.textContent = 'Computer';
+	pVCButton.className = 'selectedButton';
+});
+
+// start button
+//turns on game, disables button after click, displays text
+startButton.addEventListener('click', () => {
+	gameOn = true;
+	startButton.disabled = true;
+	interval = setInterval(timer, 1000);
+	//events to place mark for clicks in boxes
+	cell0.addEventListener('click', mark);
+	cell1.addEventListener('click', mark);
+	cell2.addEventListener('click', mark);
+	cell3.addEventListener('click', mark);
+	cell4.addEventListener('click', mark);
+	cell5.addEventListener('click', mark);
+	cell6.addEventListener('click', mark);
+	cell7.addEventListener('click', mark);
+	cell8.addEventListener('click', mark);
+	playerTurn.textContent = "Player X's turn";
+});
+
+resetButton.addEventListener('click', reset);
+
+//used to mark squares, also change player turn text content
+//if logic keeps track of turns and keeps clicking on an already selected square from overwriting, and works when start button has been initiated
+function mark(event) {
+	if (event.target.textContent === '') {
+		if (gameOn === true && move % 2 === 0) {
+			event.target.textContent = 'x';
+			move = move + 1;
+			playerTurn.textContent = "Player O's turn";
+			currentPlayer = playerXName.textContent;
+			winCheck(event.target.textContent);
+		} else if (gameOn === true && (move + 2) % 2 === 1 && computer === true) {
+//computer guess logic
+			comGuess();
+		} else if (gameOn === true && (move + 2) % 2 === 1 && computer === false) {
+			event.target.textContent = 'o';
+			move = move + 1;
+			playerTurn.textContent = "Player X's turn";
+			currentPlayer = playerOName.textContent;
+			winCheck(event.target.textContent);
+		}
+	} else if (event.target.textContent === 'x' || event.target.textContent === 'o') {
+		playerTurn.textContent = 'Please select an empty cell';
+	}
+}
+
+//reset
+function reset() {
+	gameOn = false;
+	//--------return to cells to empty
+	cell0.textContent = '';
+	cell1.textContent = '';
+	cell2.textContent = '';
+	cell3.textContent = '';
+	cell4.textContent = '';
+	cell5.textContent = '';
+	cell6.textContent = '';
+	cell7.textContent = '';
+	cell8.textContent = '';
+	//--------remove winning class
+	cell0.className = '';
+	cell1.className = '';
+	cell2.className = '';
+	cell3.className = '';
+	cell4.className = '';
+	cell5.className = '';
+	cell6.className = '';
+	cell7.className = '';
+	cell8.className = '';
```

Comment:
Like on lines 89-97 you could iterate over your `cellLookup` object to do all of this much more concisely, and save yourself a few lines of code.